packages/slate/src/interfaces/text.ts

```diff
@@ -63,6 +60,14 @@ export const Text = {
     return Array.isArray(value) && (value.length === 0 || Text.isText(value[0]))
   },
 
+  /**
+   * Check if some props are a partial of element.
+   */
+
+  isTextProps(props: any): props is Partial<Text> {
+    return !!props.children
+  },
```

Comment:
I'm making a couple fixes and will push a commit soon with this fix too, but it should be something like this:

```suggestion
  isTextProps(props: any): props is Partial<Text> {
    return (props as Partial<Text>).text !== undefined;
  },
```